### ISSUE TITLE
Movers: description text for horizontal movement is incorrect with multiple selection

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#25158]
**Describe the bug**
Noticed while working on https://github.com/WordPress/gutenberg/pull/24894

Some blocks can be moved horizontally, for example the buttons in the "Buttons" block and the Social icons.

In this case, the mover buttons aria-label (and tooltip) correctly says "Move right" or "Move left" instead of down and up.

However, there's also some descriptive text used for the `aria-describedby` attribute that provides more info on the position etc. This descriptive text is correct when there's only one block selected. See in the screenshot below, with just one button selected the text is: "Move Button block from position 1 right to position 2":

<img width="785" alt="Screenshot 2020-09-08 at 17 12 15" src="https://user-images.githubusercontent.com/1682452/92495807-a7a44c00-f1f7-11ea-88e2-6334f0f8307e.png">

Instead, the text is _not_ correct when there's more than one block selected. See in the screenshot below, with two buttons selected the text is: "Move 2 blocks from position 1 down by one place":

<img width="798" alt="Screenshot 2020-09-08 at 17 10 47" src="https://user-images.githubusercontent.com/1682452/92496118-08338900-f1f8-11ea-8f44-74eb19e8a1da.png">

Where "down" is wrong and it should be "right".

Similarly, when moving to left. There's some logic in place to build these strings and it should be reviewed. Also, checking what happens with RTL languages is important.

**Editor version (please complete the following information):**
- WordPress version: 5.6 trunk
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? both
- If the Gutenberg plugin is installed, which version is it? 8.9.2

**Comment by thisissandip** [comment]
PR added: https://github.com/WordPress/gutenberg/pull/32337
